package.json

```diff
@@ -8,7 +8,7 @@
     "local-onsuccess": "RUN_ON_SUCCESS=true node -e 'import(\"./src/index.js\").then(index => index.default()).then(events => events.onSuccess());'",
     "lint": "eslint 'src/**/*.js'",
     "format": "prettier --write 'src/**/*.js'",
-    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js --collect-coverage",
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js --collect-coverage --maxWorkers=1",
```

Comment:
We're currently not mocking express, so running multiple tests in parallel means we get domain conflicts. Adding this option runs tests sequentially (~4s total time at the moment). This could be a good follow-up issue to look into